osh.spec

```diff
@@ -234,15 +234,19 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
```

Comment:
Note that you do not need to exclude this file. What I pointed is the error "E: non-executable-script". This means that the file has `#!...` but not executable bit. This is a confusing state.
The solution is **either** set executable bit **or** remove the first line with `#!`.

Reply:
True.  My proposal was to remove them completely because we always use the scripts from a git repository.  So they do not need to be packaged at all: https://github.com/openscanhub/openscanhub/pull/261#discussion_r1576198499